### PR TITLE
Remove duplicate property key

### PR DESCRIPTION
**Pull request by nroduit**
Remove the duplicate key: `itcavantgarde`

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [ ] Commit message must refer to an issue
- [ ] Change must be properly reviewed

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/6890/head:pull/6890` \
`$ git checkout pull/6890`

Update a local copy of the PR: \
`$ git checkout pull/6890` \
`$ git pull https://git.openjdk.java.net/jdk pull/6890/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6890`

View PR using the GUI difftool: \
`$ git pr show -t 6890`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/6890.diff">https://git.openjdk.java.net/jdk/pull/6890.diff</a>

</details>

### CODE REVIEW

**Comment by TheShermanTanker** [comment]
Comment to keep alive

File: src/java.desktop/share/conf/psfontj2d.properties [hunk 1]
```diff
@@ -59,7 +59,6 @@ avantgarde_book_oblique=avantgarde_book_oblique
 avantgarde_demi_oblique=avantgarde_demi_oblique
 #
 itcavantgarde=avantgarde_book
-itcavantgarde=avantgarde_book
```
Comment (mrserb) [inline src/java.desktop/share/conf/psfontj2d.properties:62]: @prrace is it possible that this is a typo and itcavantgarde_book should be used instead?

Comment (mrserb) [reply src/java.desktop/share/conf/psfontj2d.properties:62]: @prrace  Any idea about this code?

**Comment by mrserb** [comment]
@prrace Any idea about this code?

**Pull request closed without merging**